.gitignore

```diff
@@ -15,7 +15,7 @@ dist
 *.js
 *.d.ts
 docs
-core
+/core
```

Comment:
😠  stupid that you had to do this.

Reply:
Agreed. I think we are going to have to build the package with a separate package.json in a different directory. @aneyzberg and I looked at it, but we backed out. It is not super quick and causes other issues that take time to resolve (like rebuilding the package.json with the correct stuff and in the correct paths)